Plots/Colors/CB_Height.py

```diff
@@ -0,0 +1,129 @@
+"""
+CB_Height.py
+================
+
+This script illustrates multiple color schemes for color maps which will allow for those
+impacted by color blindness to see visualizations. Using rainbow color schemes is also
+a poor choice in color scheme for images that may be transferred to a black and white 
```

Comment:
Can we make  a stronger case for why to avoid the rainbow? See the three reasons given in the Moreland paper: https://doi.org/10.2352/ISSN.2470-1173.2016.16.HVEI-133

Reply:
@clyne I include the three reasons in the figure 1 descriptions, but can move them here instead and change that description if you think that would be the best route

Reply:
OK. Looks good.